### ISSUE TITLE
Add ability to run a single test or a group

### ISSUE THREAD

**Issue by rn** [linuxkit/rtf#13]
With the original python based implementation one could run a single test or group of tests. Very useful when debugging and it would be nice if we could do:
```
rtf run foo.bar.baz
```

**Comment by dave-tucker** [comment]
You can as of https://github.com/linuxkit/rtf/commit/06358c6a2514cbf304b370cbc395aa536d555f4e

**Comment by rneugeba** [comment]
ah, then the help message needs updating:
```
# rtf run --help
A brief description of your command

Usage:
  rtf run [flags]

Flags:
  -h, --help   help for run

Global Flags:
  -c, --casedir string        Directory containing cases (default "cases")
  -e, --envVars stringSlice   Add a environment variable (multiple uses allowed)
  -x, --extra                 Add extra debug info to log files
  -l, --labels string         Labels to apply (comma separated)
      --logger string         <host>:<port> of a socket to log stdout to
  -r, --resultdir string      Directory to place results in (default "_results")
  -v, --verbose count[=-1]    Increase verbosity level
```

**Comment by dave-tucker** [comment]
ah so it does!

**Comment by dave-tucker** [comment]
Fixed in #17
